src/logger.js

```diff
@@ -20,6 +20,7 @@ if (ipcRenderer) {
   userDataPath = app.getPath('userData');
   isDevMode = process.argv[2] === '--dev';
 }
+console.log('user data path:', userDataPath);
```

Comment:
Let's remove this before merging, if you don't mind.

Reply:
ugghh I removed it yesterday but forgot to push! ready to go now.